packages/core/src/components/dropdown/test/default/index.html

```diff
@@ -20,9 +20,14 @@
       helper="Helper text"
       size="lg"
       open-direction="auto"
+      data-testid="tds-dropdown-testid"
```

Comment:
How come using `data-testid`?

Reply:
There is a way of getting that element using 

```
const dropdown = page.getByTestId('tds-dropdown-testid');
```
and the frameworks recommend to use that for getting a specific element instead of `page.locate('tds-dropdown');`